### ISSUE TITLE
notebook list variable won't load in data viewer during debug

### ISSUE THREAD

**Issue by stephenfrost** [microsoft/vscode-data-wrangler#138]
Type: <b>Bug</b>

# Behaviour

## Expected vs. Actual

During debug of a notebook cell, when I click arrow to show variable snapshot in data viewer, I expect a column of list items. Instead, the "Loading data frame" action popup shows indefinitely and data is never shown in data wrangler tab that opens up.

## Steps to reproduce:

1. In one notebook cell that defines a function, create a list.
2. Set a breakpoint on the return statement of this funciton.
3. Start debug of other cell that calls this function.
4. When execution stops at the breakpoint, show bottom pane, Jupyter tab.
5. Select arrow to left of list variable.
The data wrangler data viewer tab opens and "Loading data frame" popup appears, but the data is never displayed.

# Diagnostic data

-   Jupyter extension version: 2024.1.1
-   Python extension version: 2024.0.1
-   .NET Install Tool for Extension Authors extension version: 2.0.1
-   Python package dependencies:

```json
N/A
```

Extension version: 0.26.0
VS Code version: Code 1.86.2 (903b1e9d8990623e3d7da1df3d33db3e42d80eda, 2024-02-13T19:40:56.878Z)
OS version: Windows_NT x64 10.0.19045
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-10850H CPU @ 2.70GHz (12 x 2712)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|31.59GB (16.45GB free)|
|Process Argv|--crash-reporter-id f79e1be6-a462-418e-8c4e-0d01dc66d017|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscoreces:30445986
vscod805cf:30301675
binariesv615:30325510
vsaa593cf:30376535
py29gd2263:30899288
c4g48928:30535728
azure-dev_surveyone:30548225
a9j8j154:30646983
962ge761:30959799
pythongtdpath:30769146
welcomedialog:30910333
pythonidxpt:30866567
pythonnoceb:30805159
asynctok:30898717
pythontestfixt:30902429
pythonregdiag2:30936856
pyreplss1:30897532
pythonmypyd1:30879173
pythoncet0:30885854
pythontbext0:30879054
dsvsc016:30899300
dsvsc017:30899301
dsvsc018:30899302
fegfb526:30961430
7j2b6412:30967274
bg6jg535:30958357

```

</details>

**Comment by pwang347** [comment]
Hi @stephenfrost! Thanks for opening this issue. This is currently known behaviour (see https://github.com/microsoft/vscode-data-wrangler/issues/131, https://github.com/microsoft/vscode-data-wrangler/issues/49) and we are looking into ways to work around it. Will provide any updates here.

**Comment by pwang347** [comment]
Hi @stephenfrost this is now fixed in pre-release version 0.29.8. Thanks for opening this issue!
